### PR TITLE
Edit draft collaboration post and publish.

### PR DESCRIPTION
**Pull request by studiospring**

### CODE REVIEW

**Comment by simonv3** [comment]
@studiospring maybe you forgot to add the new file?

**Comment by studiospring** [comment]
@simonv3 Thanks! :P

**Comment by simonv3** [comment]
Awesome. Looks great! Happy to merge when you are!

File: _posts/2017-03-27-osd-needs-better-collaboration-tools.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Despite the rising awareness and acceptance of UX design, in particular on the w
 
 In this post, I will argue the need for better communication and collaboration platforms by looking at some of the fundamental characteristics of open source design and development and propose some starting points.
 
-The goal of OSD is to improve the user experience of open source software. Because of the nature of open source software development, to achieve this goal, certain critical infrastructure is necessary. I am talking about a way for designers to share and build upon ideas, interact, reach consensus and collaborate with other designers and all the other stakeholders involved in the creation of software.
+The goal of OSD is to improve the user experience of open source software. Because open source software development is distributed, certain critical infrastructure is necessary. I am talking about a way for designers to share and build upon ideas, interact, reach consensus and collaborate with other designers and all the other stakeholders involved in the creation of software.
```
Comment (jancborchardt) [inline _posts/2017-03-27-osd-needs-better-collaboration-tools.md]: @studiospring Just a nitpick: Could you change the »OSD« to »Open Source Design«? Acronyms are quite difficult to understand for newcomers, and we always want to be welcoming to new people. :)

Comment (grahamperrin) [inline _posts/2017-03-27-osd-needs-better-collaboration-tools.md]: Also @studiospring _if you like_, an invitation to readers of the article to discuss it under https://discourse.opensourcedesign.net/t/open-source-design-needs-better-collaboration-tools/22

Side note: feel free to edit that topic and if you wish to make it your own, the system will literally allow you change ownership.

Comment (jancborchardt) [inline _posts/2017-03-27-osd-needs-better-collaboration-tools.md]: @grahamperrin good stuff on kickstarting a discussion on an article! :) I think there might even be a way to embed the Discourse discussion thread on the site as some sort of comment form.

**Pull request merged**